packages/subgraph/src/entities/TokenBalance.ts

```diff
@@ -27,11 +27,17 @@ export function getTokenBalance(_token: Address, _owner: Address): TokenBalance
   }
 }
 
-export function updateBalance(_event: ethereum.Event, _from: Address, _to: Address, _value: BigInt): void {
+export function updateBalance(
+  _event: ethereum.Event,
+  _from: Address,
+  _to: Address,
+  _value: BigInt
+): void {
   let tokenAddress = _event.address;
   let token = getToken(tokenAddress);
 
-  if (_from.toHexString() == ZERO_ADDRESS) { // mint
+  if (_from.toHexString() == ZERO_ADDRESS) {
+    // mint
```

Comment:
Was this automated by the editor or it’s that you like it better this way?

Reply:
This is automatically done by prettier. We have a config file in the root dir:
https://github.com/liquity/dev/blob/main/.prettierrc.json

Reply:
Oh, nice! I’ll try to remember to run it always before pushing (maybe I’ll try to add a git hook next time, as I’m kind of forgetful, haha).
Thanks!